### PR TITLE
Production: Ai chat release partial enable production (#795)

### PR DESCRIPTION
**Pull request by petemill**
https://github.com/brave/brave-variations/pull/795 on `production`

### CODE REVIEW

**Comment by simonhong** [comment]
Just force pushed with same changes to make run `Test` job again.

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers. QA has verified the PR on `1.60.110 Chromium: 119.0.6045.105` using the `staging` server as per https://github.com/brave/brave-variations/pull/795#issuecomment-1789763617.

**Pull request merged**
